docker/Jenkinsfile-build-docker

```diff
@@ -105,7 +105,7 @@ ansiColor('xterm') {
         }
         // add python 16 manually, we dont need all JAVA_16xPYTHON_VERSIONS and JAVA_16xR_VERSION  images
         // to be able run Java 16 on nightly
-        componentJdkImages += "dev-python-2.7-jdk-16" // FIXME can be deleted when Java 16 not run in NIGHTLY_REPEATED_STAGES
+        componentJdkImages += "dev-python-3.6-jdk-16" // FIXME can be deleted when Java 16 not run in NIGHTLY_REPEATED_STAGES
```

Comment:
This line about java 16 can be entirely deleted, since we are not testing Java 16 anymore in nightly